collections/ordering.go

```diff
@@ -0,0 +1,16 @@
+package collections
+
+import "errors"
+
+var ErrInvalidOrderBy = errors.New("invalid order_by")
```

Comment:
Following on from the linter discussion today - should probably add GoDoc for public types.